src/Controls/samples/Controls.Sample/Pages/MainPage.cs

```diff
@@ -22,149 +24,25 @@ public MainPage(MainPageViewModel viewModel)
 
 		void SetupMauiLayout()
 		{
-			const string loremIpsum =
```

Comment:
I'm assuming you know this got checked in :-)

You could also just create your own page for this like Rachel did here
https://github.com/dotnet/maui/pull/679

And then at some point we'll implement navigation/tabbing or some futuristic form of switching pages

Reply:
Yup, the samples will be fixed back up to their normal state!